js/index.js

```diff
@@ -0,0 +1,39 @@
+const bodyElement = document.querySelector('[data-js="body"]');
+
+const headerElement = document.querySelector('[data-js="header"]');
+
+const navbarElement = document.querySelector('[data-js="navbar"]');
+
+const buttonToggle = document.querySelector('[data-js="toggle-button"]');
+
+buttonToggle?.addEventListener("click", () => {
+  bodyElement.classList.toggle("dark");
+  headerElement.classList.toggle("dark");
+  navbarElement.classList.toggle("dark");
+
+  if (bodyElement.classList.contains("dark")) {
+    buttonToggle.textContent = "Light Mode";
+  } else {
+    buttonToggle.textContent = "Dark Mode";
+  }
+});
+
+const buttonBookmark = document.querySelector('[data-js="bookmark-button"]');
+const bookmarkIcon = document.querySelector('[data-js="bookmark-icon"]');
+
+buttonBookmark?.addEventListener("click", () => {
+  bookmarkIcon.classList.toggle("active");
+});
+
+const buttonAnswer = document.querySelector('[data-js="answer-button"]');
+const answer = document.querySelector('[data-js="answer"]');
+
+buttonAnswer?.addEventListener("click", () => {
```

Comment:
What does the question mark do? Is it a typo?

Reply:
Mit dem Fragezeichen kann man mehrere unterschiedliche JS Funktionen in einer JS-Datei schreiben, das sollte man aber eigentlich nicht machen und war nur eine vorübergehende Lösung, weil wir damals noch nicht die JS structure session hatten :)